src/Compilers/CSharp/Portable/Symbols/NullableAnnotationExtensions.cs

```diff
@@ -52,5 +53,28 @@ internal static CodeAnalysis.NullableAnnotation ToPublicAnnotation(this Nullable
             Debug.Assert((CodeAnalysis.NullableAnnotation)(NullableAnnotation.Oblivious + 1) == CodeAnalysis.NullableAnnotation.Disabled);
             return (CodeAnalysis.NullableAnnotation)annotation + 1;
         }
+
+        internal static NullabilityInfo ToNullabilityInfo(this CodeAnalysis.NullableAnnotation annotation, TypeSymbol type)
+        {
+            if (annotation == CodeAnalysis.NullableAnnotation.NotApplicable)
+            {
+                return default;
+            }
+
+            NullableAnnotation internalAnnotation = (NullableAnnotation)annotation - 1;
```

Comment:
>NullableAnnotation internalAnnotation = (NullableAnnotation)annotation - 1; [](start = 12, length = 75)

Consider using a `switch` instead, to avoid depending on the actual values of the internal enum.